test/games/strategy/engine/message/EndPointTest.java

```diff
@@ -6,10 +6,9 @@
 import junit.framework.TestCase;
 
 public class EndPointTest extends TestCase {
-  @SuppressWarnings("rawtypes")
   public void testEndPoint() {
     final EndPoint endPoint = new EndPoint("", Comparator.class, false);
-    endPoint.addImplementor(new Comparator() {
+    endPoint.addImplementor(new Comparator<Object>() {
```

Comment:
That sounds like there may be a bug, should work with either EndPoint or Object.